vmdb/app/views/ops/_rbac_tag_box.html.haml

```diff
@@ -3,7 +3,7 @@
   %table.style1
     %tr
       %td.key
-        = session[:customer_name]
+        = current_brand.customer_name
```

Comment:
I'm mixed on this one. We have so many globals.
Was aiming to move a bunch of the current_user gook into an object so we don't have all these globals, but then again, this is very verbose.